### PR TITLE
feat: add more voc mutations to config

### PR DESCRIPTION
**Pull request by alethomas**
add all current voc mutations based on rki mention and remove duplication

https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/DESH/Bericht_VOC_2021-07-14.pdf?__blob=publicationFile

### CODE REVIEW

File: config/config.yaml [hunk 1]
```diff
@@ -160,11 +150,36 @@ read_lineage_call:
 # variants of concern (protein name -> variants)
 voc:
   S:
-    - N501Y
-    - E484K
-    - K417N
+    - L18F
```
Comment (thomasbtf) [inline config/config.yaml:153]: I think we should place this before the benchmarking block, not in it. With benchmarking block I mean above the large comment in line 88

Comment (alethomas) [reply config/config.yaml:153]: so also the duplication was on purpose?

File: config/config.yaml [hunk 2]
```diff
@@ -156,16 +181,6 @@ read_lineage_call:
...
     - 250
 
-
-# variants of concern (protein name -> variants)
-voc:
-  S:
-    - N501Y
-    - E484K
-    - K417N
-    - H69_V70del
```
Comment (alethomas) [inline config/config.yaml:166]: Is this needed? @thomasbtf

Comment (thomasbtf) [reply config/config.yaml:166]: No, I don't think so :)

**Review by thomasbtf — APPROVED** [review]

**Pull request merged**